src/Equinor.ProCoSys.IPO.Query/GetMcPkgsUnderCommPkgInProject/GetMcPkgsInProjectQueryHandler.cs

```diff
@@ -7,28 +7,28 @@
 using MediatR;
 using ServiceResult;
 
-namespace Equinor.ProCoSys.IPO.Query.GetMcPkgsInProject
+namespace Equinor.ProCoSys.IPO.Query.GetMcPkgsUnderCommPkgInProject
 {
-    public class GetMcPkgsInProjectQueryHandler : IRequestHandler<GetMcPkgsInProjectQuery, Result<List<ProCoSysMcPkgDto>>>
+    public class GetMcPkgsUnderCommPkgInProjectQueryHandler : IRequestHandler<GetMcPkgsUnderCommPkgInProjectQuery, Result<List<ProCoSysMcPkgDto>>>
```

Comment:
GetMcPkgsInProjectByCommPkgNo? Feel like that's closer to our usual naming conventions but just a suggestion:-)